mutation_number,failing_functions
0,"[tests/test_auth_methods.py::test_login_all_methods_fail_raises_error]"
1,"[tests/test_email_utils.py::test_extract_recipients_valueerror_on_multiple_resent_message]"
2,"[tests/test_email_utils.py::test_parse_address, tests/test_email_utils.py::test_quote_address, tests/test_email_utils.py::test_extract_sender]"
3,"[tests/test_commands.py::test_expn_smtputf8_supported]"
4,"[tests/test_errors.py::test_raise_smtp_recipient_refused, tests/test_sendmail.py::test_sendmail_with_rcpt_option]"
5,"[tests/test_auth_methods.py::test_login_tries_all_methods, tests/test_auth_methods.py::test_auth_login_success]"
6,"[tests/test_protocol.py::test_protocol_writer_connected_check_on_start_tls]"
7,"[tests/test_esmtp_utils.py::test_basic_extension_parsing]"
8,"[tests/test_commands.py::test_rcpt_error]"
9,"[tests/test_api.py::test_send_with_login]"
10,"[tests/test_email_utils.py::test_extract_sender_resent_message]"
11,"[tests/test_api.py::test_send_with_login]"
12,"[tests/test_api.py::test_send_with_login]"
13,"[tests/test_asyncio.py::test_many_commands_with_gather, tests/test_commands.py::test_vrfy_ok, tests/test_commands.py::test_vrfy_with_blank_address]"
14,"[tests/test_commands.py::test_expn_smtputf8_supported, tests/test_commands.py::test_expn_smtputf8_not_supported]"
15,"[tests/test_main.py::test_command_line_send]"
16,"[tests/test_api.py::test_send_with_login]"
17,"[tests/test_asyncio.py::test_many_commands_with_gather, tests/test_commands.py::test_vrfy_ok, tests/test_commands.py::test_vrfy_with_blank_address, tests/test_commands.py::test_vrfy_smtputf8_supported]"
18,"[tests/test_main.py::test_command_line_send]"
19,"[tests/test_api.py::test_send_with_login]"
20,"[tests/test_commands.py::test_rcpt_ok, tests/test_commands.py::test_rcpt_error]"
21,"[tests/test_asyncio.py::test_many_commands_with_gather, tests/test_commands.py::test_help_ok, tests/test_commands.py::test_help_error]"
22,"[tests/test_auth_methods.py::test_login_tries_all_methods, tests/test_auth_methods.py::test_auth_crammd5_success]"
23,"[tests/test_asyncio.py::test_many_commands_with_gather, tests/test_commands.py::test_noop_ok, tests/test_commands.py::test_noop_error]"
24,"[tests/test_config.py::test_default_port_on_connect]"
25,"[tests/test_asyncio.py::test_many_commands_with_gather, tests/test_commands.py::test_help_ok, tests/test_commands.py::test_help_error]"
26,"[tests/test_auth_utils.py::test_auth_crammd5_verify_bytes, tests/test_auth_utils.py::test_auth_crammd5_verify_str]"
27,"[tests/test_asyncio.py::test_many_commands_with_gather, tests/test_commands.py::test_expn_ok, tests/test_commands.py::test_expn_error, tests/test_commands.py::test_expn_smtputf8_supported]"
28,"[tests/test_response.py::test_response_repr]"
29,"[tests/test_commands.py::test_vrfy_smtputf8_supported]"
30,"[tests/test_api.py::test_send_with_login]"
31,"[tests/test_commands.py::test_vrfy_smtputf8_supported, tests/test_commands.py::test_vrfy_smtputf8_not_supported]"
32,"[tests/test_sendmail.py::test_send_message_with_formataddr]"
33,"[tests/test_config.py::test_local_hostname_newline_raises_error]"
